tests_bucklescript/__tests__/snapshots.re

```diff
@@ -35,7 +35,7 @@ let run_ppx = (path, opts) => {
     ++ (win ? " - -o -" : " /dev/stdin /dev/stdout")
     ++ " | "
     ++ refmt
-    ++ " --parse binary --print ml --interface false",
+    ++ " --parse binary --interface false",
```

Comment:
So far we're outputting OCaml output. Is there a way to snapshot Reason output?

Reply:
Yes it should be reason. Let me check.

Reply:
Actually with this change it prints in Reason. I just didn't update the snapshots yet (in a new commit)